readinglist/sessions/__init__.py

```diff
@@ -0,0 +1,24 @@
+class SessionStorageBase(object):
```

Comment:
For know we don't use session with a ``session_id`` it seems a bit to much for what we are planning to do. But if we want to add session and store the session_id in a cookie, we will definitely consider using this.